src/main/java/edu/cornell/kfs/rass/batch/xml/RassXmlAgencyEntry.java

```diff
@@ -0,0 +1,130 @@
+package edu.cornell.kfs.rass.batch.xml;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.adapters.NormalizedStringAdapter;
+import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
+
+import org.apache.commons.lang3.StringUtils;
+import org.kuali.kfs.sys.KFSConstants;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlRootElement(name = "Agency", namespace = StringUtils.EMPTY)
+public class RassXmlAgencyEntry {
+    
+    @XmlElement(name = "Number", namespace = StringUtils.EMPTY, required = true)
+    @XmlJavaTypeAdapter(NormalizedStringAdapter.class)
+    private String number;
+    
+    @XmlElement(name = "Reporting_Name", namespace = StringUtils.EMPTY, required = true)
+    @XmlJavaTypeAdapter(NormalizedStringAdapter.class)
+    private String reportingName;
+    
+    @XmlElement(name = "Full_Name", namespace = StringUtils.EMPTY, required = true)
+    @XmlJavaTypeAdapter(NormalizedStringAdapter.class)
+    private String fullName;
+    
+    @XmlElement(name = "Type_Code", namespace = StringUtils.EMPTY, required = true)
+    @XmlJavaTypeAdapter(NormalizedStringAdapter.class)
+    private String typeCode;
+    
+    @XmlElement(name = "Reports_to_Agency_Number", namespace = StringUtils.EMPTY, required = true)
```

Comment:
since the required attribute doesn't buy anything here and we don't know what RASS considers required or not, I am going to remove the required annotation everywhere, thanks for noticing